HelpSource/scdoc.css

```diff
@@ -15,7 +15,6 @@ html {
 }
 
 div.contents {
-    margin: 1em;
```

Comment:
sorry, i think i commented on the wrong line. this is what's making it cramped. i recommend leaving this in.